frontend/server/src/Controllers/Session.php

```diff
@@ -49,10 +49,7 @@ public static function getSessionManagerInstance(): \OmegaUp\SessionManager {
     }
 
     public static function getFacebookLoginUrl(): string {
-        $scopedFacebook = new ScopedFacebook();
-        $facebook = $scopedFacebook->facebook;
-
-        $helper = $facebook->getRedirectLoginHelper();
+        $helper = ((new ScopedFacebook())->facebook)->getRedirectLoginHelper();
```

Comment:
oh espera, me refería a que accedieras `$scopedFacebook->facebook` en vez de crear `$facebook`. aquí nuevamente estás cerrando la sesión inmediatamente porque no estás asignando el `ScopedFacebook` en ningún lugar. entonces esto debería quedar

```suggestion
        $scopedFacebook = new ScopedFacebook();
        $helper = $scopedFacebook->facebook->getRedirectLoginHelper();
```

Reply:
Ohh, ya entiendo.